Source/Core/PolylineVolumeOutlineGeometry.js

```diff
@@ -133,12 +135,8 @@ define([
         var shape = options.shapePositions;
 
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(positions)) {
-            throw new DeveloperError('options.polylinePositions is required.');
-        }
-        if (!defined(shape)) {
-            throw new DeveloperError('options.shapePositions is required.');
-        }
+        Check.defined('positions', positions);
```

Comment:
`Check.defined('options.polylinePositions', positions);`

The string should always be the param name passed in